### DISCUSSION TITLE
公告及文档

### DISCUSSION THREAD

**Discussion by RekinChiang** [AlistGo/alist#4701] [General]
近期2次变动: HTTP/HTTPS端口分离, HTTP/HTTPS配置参数发生变动. 都导致Docker自动更新容器后, 无法正常访问, 文档也没及时更新. 

Alist现在已经有较大的用户群, 每一个特性/配置的变更调整都会对不少用户产生影响. 

作者以及所有贡献者能否在对相关特性/配置调整前在Discussions发个公告并及时更新文档?

**Comment by xhofe** [comment]
好的 文档已经更新，我稍后发个公告。
